From 765a380b4ed11e2210290633c5260ae53bebc5a4 Mon Sep 17 00:00:00 2001
Message-Id: <765a380b4ed11e2210290633c5260ae53bebc5a4.1657696019.git.geliang.tang@suse.com>
From: Geliang Tang <geliang.tang@suse.com>
Date: Wed, 13 Jul 2022 15:06:53 +0800
Subject: [PATCH] Revert "Linux 5.10.130"
Patch-mainline: Never, revert stable version change
References: bsn#83
Modified-by-SEL: No

This reverts commit 26ae9c361414418ed02d6e97b3d0c8eaa93be355.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index b0a35378803d..7d52cee37488 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 10
-SUBLEVEL = 130
+SUBLEVEL = 129
 EXTRAVERSION =
 NAME = Dare mighty things
 
-- 
2.34.1

